integration_tests/integration/bookedtoday/unexpectedArrivals/searchForExistingRecord.spec.ts

```diff
@@ -16,23 +16,20 @@ context('Unexpected arrivals - Search for existing record spec', () => {
     SearchForExistingPage.goTo()
   })
 
-  it('Search with no details', () => {
+  it('Search without first name, last name or date of birth', () => {
     const searchPage = Page.verifyOnPage(SearchForExistingPage)
     searchPage.search().click()
-    searchPage.hasError("You must search using either the prisoner's last name, prison number or PNC Number")
-  })
-  it('Enter first name only', () => {
-    const searchPage = Page.verifyOnPage(SearchForExistingPage)
-    searchPage.firstName().type('James')
-    searchPage.search().click()
-    searchPage.hasError('Enter a last name')
+    searchPage.hasError(
+      "You must search using either the prisoner's last name and date of birth, prison number or PNC Number"
+    )
   })
 
-  it('Enter partial date only', () => {
+  it('Enter partial date of birth', () => {
     const searchPage = Page.verifyOnPage(SearchForExistingPage)
+    searchPage.firstName().type('James')
+    searchPage.lastName().type('Smyth')
     searchPage.day().type('01')
     searchPage.search().click()
-    searchPage.hasError("You must search using either the prisoner's last name, prison number or PNC Number")
     searchPage.hasError('Date of birth must include a month and year')
   })
 
```

Comment:
Your new specs cover most of these already?
* Redirects to no match page
* Redirects to single match page
* Redirects to multiple match page

You can delete these tests from here?

Reply:
In fact maybe just remove this spec entirely?

Reply:
Yes, I agree it would make sense to test the validation message is shown on the search page but probably only need to do this once and can check as part of the end to end flow for the journey